scripts/linters/general_purpose_linter.py

```diff
@@ -268,6 +268,14 @@
             'core/templates/pages/library-page/search-bar/'
             'search-bar.component.spec.ts'),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'import \{.*\} from \'lodash\''),
```

Comment:
Isn't this a bit overkill? I mean here just regex is enough looking at custom eslint checks they look a lot complicated, also do we have any links to wiki (external) how to write them?

This check is needed so that the tree shaking (webpack's way to only include parts of library which are actually needed) works properly for lodash.

Reply:
Maybe we can have custom eslint rule that would take a list of regexes that are forbidden? Or have a custom eslint rules that we can inherit from to write custom regex rules?

Reply:
We will handle this seperetly, once we will have a structure!